src/main/java/reactor/netty/resources/ConnectionProvider.java

```diff
@@ -100,14 +111,7 @@ static ConnectionProvider elastic(String name) {
 	 * {@link Connection}
 	 */
 	static ConnectionProvider elastic(String name, @Nullable Duration maxIdleTime, @Nullable Duration maxLifeTime) {
-		return new PooledConnectionProvider(name,
-				(allocator, destroyHandler, evictionPredicate) ->
-				                PoolBuilder.from(allocator)
-				                           .destroyHandler(destroyHandler)
-				                           .evictionPredicate(evictionPredicate
-				                                   .or((poolable, meta) -> (maxIdleTime != null && meta.idleTime() >= maxIdleTime.toMillis())
-				                                           || (maxLifeTime != null && meta.lifeTime() >= maxLifeTime.toMillis())))
-				                           .fifo());
+		return Builder.newInstance(name).maxIdleTime(maxIdleTime).maxLifeTime(maxLifeTime).build();
```

Comment:
You need to specify that it is elastic `.maxConnections(MAX_CONNECTIONS_ELASTIC).acquireTimeout(ACQUIRE_TIMEOUT_NEVER_WAIT)`